src/Kanbas/Courses/Quizzes/Editor/Details/index.tsx

```diff
@@ -1,11 +1,245 @@
+                            'bold italic forecolor | alignleft aligncenter ' +
+                            'alignright alignjustify | bullist numlist outdent indent | ' +
+                            'removeformat | help',
+                        content_style: 'body { font-family:Helvetica,Arial,sans-serif; font-size:14px }'
+                    }}
+                />
+            </Form.Group>
+            <div>
+                <Form.Group className="row mb-3" controlId="formQuizType">
+                    <div className="col-3">
+                        <Form.Label className="float-end mt-1">
+                            Quiz Type
+                        </Form.Label>
+                    </div>
+                    <div className="col-6 mr-auto">
+                        <Form.Select
+                            aria-label="Quiz Type"
+                            value={quizDetails.quizType}
+                            onChange={(e) => dispatch(setQuiz({
+                                ...quizDetails, quizType: e.target.value
+                            }))}
+                        >
+                            {quizTypes.map((key, index) => (
+                                <option key={index} value={eQuizType[key as keyof typeof eQuizType]}>
+                                    {key}
+                                </option>
+                            ))}
+                        </Form.Select>
+                    </div>
+                </Form.Group>
+
+                <Form.Group className="row mb-3" controlId="formAssignmentGroup">
+                    <div className="col-3">
+                        <Form.Label className="float-end mt-1">
+                            Assignment Group
+                        </Form.Label>
+                    </div>
+                    <div className="col-6 mr-auto">
+                        <Form.Select
+                            aria-label="Assignment Group"
+                            value={quizDetails.assignmentGroup}
+                            onChange={(e) => dispatch(setQuiz({
+                                ...quizDetails, assignmentGroup: e.target.value
+                            }))}
+                        >
+                            {assignmentGroup.map((key, index) => (
+                                <option key={index} value={eAssignmentGroup[key as keyof typeof eAssignmentGroup]}>
+                                    {key}
+                                </option>
+                            ))}
+                        </Form.Select>
+                    </div>
+                </Form.Group>
+
+                <Form.Group
+                    className="row mb-3"
+                    controlId="formAssignmentAssignList"
+                >
+                    <div className="col-3">
+                        <Form.Label className="float-end mt-1">Assign</Form.Label>
+                    </div>
+                    <div className="col-6 mr-auto">
```

Comment:
This card should be responsive, using similar width on different screen sizes.

Reply:
col-md-6 col-lg-4 col-xl-3 might work, but this is just a guess.

Reply:
This issue is not solved.

Reply:
This is already being responsive, that's why I didn't change